.vscode/launch.json

```diff
@@ -59,6 +59,7 @@
       "args": [
         "--hostname=0.0.0.0",
         "--port=3000",
+        "--no-cluster",
```

Comment:
Do we still want rid of that?

Reply:
No, that was an oversight on my side. Thanks for readding that.

Reply:
The revert did it, not me :)